Sheepish-Squirrels/bangazon-api-sprint1
# Able to display customers that haven't placed an order

Fixes #11 

Changes proposed in this pull request:

- wrote a query for inactive customers that gets only customers who don't have ids in orders
- added if statement in customer ctrl for getting all customers so that you can get data of inactive 
- tried to fix "404 handler" console log error to no avail
- if statement needed '===' instead of '1'. also changed else to having url condition

Steps to test:

- type 'http://127.0.0.1:3000/api/v1/customers/?active=false' in url to make sure you can see all customers who have not placed orders
- cross reference this information with the customer_ids of the Orders table to make sure customer ids shown in orders table are not displayed on this query params
- type 'http://127.0.0.1:3000/api/v1/customers' in browser to make sure all customers are displayed
